tests/phpunit/unit-tests/functions/class-llms-test-functions-user-postmeta.php

```diff
@@ -74,7 +85,7 @@ public function test_delete_user_postmeta() {
 			$i++;
 		}
 		$this->assertTrue( llms_delete_user_postmeta( $this->student_id, $this->course_id, '_test_data_to_erase', 'eraseme3' ) );
-		$this->assertEquals( array( 'eraseme1', 'eraseme2' ), llms_get_user_postmeta( $this->student_id, $this->course_id, '_test_data_to_erase', false ) );
+		$this->assertEqualSets( array( 'eraseme1', 'eraseme2' ), llms_get_user_postmeta( $this->student_id, $this->course_id, '_test_data_to_erase', false ) );
```

Comment:
TIL

Reply:
it's from the WP Core test lib: https://github.com/WordPress/wordpress-develop/blob/1ec9dc6fa46b23f7724dfa3f057e2da1a1dd3133/tests/phpunit/includes/abstract-testcase.php#L705

There's actually quite a few helpful assertions in the class. If you want homework, study them!